win32ss/user/ntuser/defwnd.c

```diff
@@ -929,12 +929,17 @@ IntDefWindowProc(
       case WM_MOUSEACTIVATE:
          if (Wnd->style & WS_CHILD)
          {
-             LONG Ret;
              HWND hwndParent;
              PWND pwndParent = IntGetParent(Wnd);
              hwndParent = pwndParent ? UserHMGetHandle(pwndParent) : NULL;
-             if (hwndParent) Ret = co_IntSendMessage(hwndParent, WM_MOUSEACTIVATE, wParam, lParam);
-             if (Ret) return (Ret);
+             if (hwndParent)
+             {
+                 LONG Ret = co_IntSendMessage(hwndParent, WM_MOUSEACTIVATE, wParam, lParam);
+                 if (Ret)
+                 {
+                     return Ret;
+                 }
+             }
```

Comment:
It seems like hwndParent==NULL should be impossible here. Can you add an assert and see if it fires?

Reply:
I tried `set DEBUGCHANNEL=+UserDefwnd && notepad` on a LiveCD:
I get a few logs like `(win32ss/user/ntuser/defwnd.c:609) trace: hwnd 000F0116 WM_SYSCOMMAND f006 1a90091`,
but I can't seem to trigger `WM_MOUSEACTIVATE`.
What are steps to actually test this case?

Reply:
You could try SendMessage. Perhaps even combined with a SetWindowLong to give the desktop window the WS_CHILD flag?